core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -144,34 +149,47 @@ popup(state){
     this.editBtn=false;
   }
 
-  onEditAppClick(){
-    this.yaml = this.yamlName;
-    this.showCncl = true;
-    this.isDiv = true;
-    this.saveButton = true;
-    this.editButton = false;
-    this.isCancel = false;
-    this.textChanged = true;
-    this.disableSave = true;
-    this.textarea = false;
+  fetchfunction(){
+    this.isLoading = true;
+    this.http.get('/jazz/services/' + this.service.id, null, this.service.id).subscribe(response => {
+      this.is_function = response.data.metadata.is_function_template;
```

Comment:
is_function is going away as we are not allowing edit. it is just view only. no readonly section. maybe dimple's PR takes care of it..